.github/workflows/tidy-modern.yml

```diff
@@ -0,0 +1,36 @@
+name: tidy-modern
+
+on:
+  push:
+  pull_request:
```

Comment:
Could this be the culprit? Empty push and pull triggers

Reply:
Oddly, I never saw a `clang-tidy` build step in #4560, although the GitHub workflow plainly ran there.

Reply:
Apparently, they changed the syntax; it's now `on: [push, pull_request]`

Once I did that, it started triggering.